teamup-client/src/app/common/login/login.validators.ts

```diff
@@ -0,0 +1,30 @@
+import {AbstractControl, ValidationErrors} from '@angular/forms';
+
+export class LoginValidators {
```

Comment:
I don`t think that we have discussed validity on username, are those validators required?